Dockerfile

```diff
@@ -520,11 +550,13 @@ COPY --from=buster-build /google-cloud-ops-agent*.deb /
 
 FROM opensuse/archive:42.3 AS sles12-build-base
 
+COPY --from=cmake-install /cmake.sh /cmake.sh
+
 RUN set -x; \
 		# The 'OSS Update' repo signature is no longer valid, so verify the checksum instead.
 		zypper --no-gpg-check refresh 'OSS Update' && \
 		(echo 'b889b4bba03074cd66ef9c0184768f4816d4ccb1fa9ec2721c5583304c5f23d0  /var/cache/zypp/raw/OSS Update/repodata/repomd.xml' | sha256sum --check) && \
-		zypper -n install git systemd autoconf automake flex libtool libcurl-devel libopenssl-devel libyajl-devel gcc gcc-c++ zlib-devel rpm-build expect cmake systemd-devel systemd-rpm-macros unzip zip wget && \
+		zypper -n install curl git systemd autoconf automake flex libtool libcurl-devel libopenssl-devel libyajl-devel gcc gcc-c++ zlib-devel rpm-build expect cmake systemd-devel systemd-rpm-macros unzip zip && \
```

Comment:
Where is `curl` used now?

Reply:
Oops, forgot to remove the lingering ones. Fixed.